src/Cargo.lock

```diff
@@ -1290,6 +1290,10 @@ dependencies = [
  "winapi-util",
 ]
 
+[[package]]
+name = "scheduler"
+version = "0.0.0"
```

Comment:
This is probably intended, but the version does not match the scheduler version `0.1.0` from `Cargo.toml`

Reply:
This wasn't intended, but is fixed now.